libs/libmilter/patches/101-pthreads-stack-size.patch

```diff
@@ -0,0 +1,34 @@
+Index: sendmail-8.15.2/libmilter/libmilter.h
```

Comment:
But... we're typically not linking against glibc, but instead MUSL or eglibc or uClibc...